internal/pkg/cli/validate.go

```diff
@@ -389,6 +488,27 @@ func dynamoTableNameValidation(val interface{}) error {
 	return nil
 }
 
+// RDS storage name: '[a-zA-Z][a-zA-Z0-9]*'
+func rdsNameValidation(val interface{}) error {
+    // This is the length constraints for DB Cluster identifier.
+    // RDS storage name will be used as logical ID, which will then be used to generate DB Cluster identifier.
+    const minRDSNameLength = 1
+    const maxRDSNameLength = 63
+
+    s, ok := val.(string)
+    if !ok {
+        return errValueNotAString
+    }
+    if len(s) < minRDSNameLength || len(s) > maxRDSNameLength {
+        return errRDSNameBadSize
+    }
+    m := rdsStorageNameRegExp.FindStringSubmatch(s)
+    if m == nil {
+        return errInvalidRDSNameCharacters
+    }
+    return nil
+}
+
```

Comment:
I love all these validations! They will prevent a lot of headaches down the road!